### PR TITLE
Add /v1/sendEmails endpoint with AWS SES backend

### PR DESCRIPTION
**Pull request by lindgrenj6**
This PR adds the `/v1/sendEmails` BOP-compatible endpoint.

Notable things include:
- leveraging the `MAILER_MODULE` template parameter, currently supporting `aws` to send emails with SES and `print` to just print the email to stdout (good for validating locally).
- leveraging the helper functions for sending errors and/or JSON responses
- introduces a few access/secret keys for AWS SES. will probably move these to secrets eventually

---

Note: this does *not* include the user lookups. I will tie those in once we have the helper functions available.

### CODE REVIEW

File: deployments/deployment.yaml [hunk 1]
```diff
@@ -31,6 +31,12 @@ objects:
         - env:
           - name: KEYCLOAK_SERVER
             value: "${KEYCLOAK_SCHEME}://${KEYCLOAK_HOST}:${KEYCLOAK_PORT}${KEYCLOAK_PATH}"
+          - name: SES_ACCESS_KEY
+            value: "${SES_ACCESS_KEY}"
+          - name: SES_SECRET_KEY
+            value: "${SES_SECRET_KEY}"
```
Comment (coderbydesign) [inline deployments/deployment.yaml:37]: Just curious if there's a reason we wouldn't want to drop these in secrets and reference them from there, now?

Comment (lindgrenj6) [reply deployments/deployment.yaml]: 100% fat finger on my part - updated to read from a secret marked as optional as to not break the current deployment.

**Review by coderbydesign — APPROVED** [review]
🎉

**Comment by lindgrenj6** [comment]
After tests pass I'm going to merge this just so the bulk is in - I'll open up a followup PR with the from-email config as well as the username lookups.

**Pull request merged**